### PR TITLE
Add missing feature for `trace.rs`

### PR DESCRIPTION
**Pull request by Renkai**
Signed-off-by: Renkai <gaelookair@gmail.com>

### What problem does this PR solve?
Some code is not actually compiled since missing configuration in `Cargo.toml`, now supplemented.

### CODE REVIEW

File: components/tikv_util/src/trace.rs [hunk 1]
```diff
@@ -18,24 +18,40 @@ pub fn encode_spans(span_sets: Vec<SpanSet>) -> impl Iterator<Item = spanpb::Spa
...
-                        s.link = spanpb::Link::Root;
-                    }
-                    Link::Parent(id) => {
-                        s.link = spanpb::Link::Parent(id);
-                    }
-                    Link::Continue(id) => {
-                        s.link = spanpb::Link::Continue(id);
-                    }
+                {
+                    use minitrace::Link;
```
Comment (BusyJay) [inline components/tikv_util/src/trace.rs:22]: Any tests?

**Comment by breeswish** [comment]
@zhongzc PTAL

File: components/tikv_util/src/trace.rs [hunk 2]
```diff
@@ -45,3 +59,67 @@ pub fn encode_spans(span_sets: Vec<SpanSet>) -> impl Iterator<Item = spanpb::Spa
...
+mod tests {
+    use kvproto::span as spanpb;
+
+    #[test]
+    fn test_encode_spans() {
+        let (guard, collector) = minitrace::trace_enable(0u32);
+        let guard2 = minitrace::new_span(1u32);
+        std::mem::drop(guard2);
+        std::mem::drop(guard);
+
```
Comment (zhongzc) [inline components/tikv_util/src/trace.rs:73]: Could you please construct `Vec<minitrace::SpanSet>` handly to be encoded, then compare the encoded `Vec<spanpb::SpanSet>` with another `Vec<spanpb::SpanSet>` constructed handly?

**Pull request merged**